gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/plan/model/Plan.java

```diff
@@ -261,6 +273,27 @@ public Plan copy() {
         };
     }
 
+    public Plan toRollback(io.gravitee.definition.model.v4.plan.Plan planDefinitionV4) {
```

Comment:
to follow what we have for Api, it should be `rollbackTo`, because you rollback to a desired state